src/app/Chart.lock

```diff
@@ -1,36 +1,36 @@
 dependencies:
 - name: common
   repository: https://arkcase.github.io/ark_helm_charts
-  version: 0.8.0-26
```

Comment:
I think the 0.9 charts are in the "new-clustering" branch.  Should you target new-clustering instead of passwords-from-secrets?